factory-hooks/post-settings-php/acsfd8+.memcache.settings.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * @file
+ * Contains caching configuration.
+ */
+use Composer\Autoload\ClassLoader;
```

Comment:
Do we want to add strict type checking to this file? I believe it came from Acquia.

Reply:
Yep!